opencj/weapons.gsc

```diff
@@ -203,4 +204,21 @@ _onSettingSlowReload(value)
     {
         self setClientCvar("perk_weapReloadMultiplier", 0.5);
     }
+}
+
+_onSettingSmallCrosshair(value)
+{
+    if(value)
```

Comment:
I would prefer the other way around, that the settings thing calls a setWeaponSpread instead of other way around